packages/bvaughn-architecture-demo/components/console/ConsoleRoot.tsx

```diff
@@ -11,6 +11,8 @@ import {
   useState,
 } from "react";
 import { TerminalContext } from "@bvaughn/src/contexts/TerminalContext";
+import useLocalStorage from "@bvaughn/src/hooks/useLocalStorage";
```

Comment:
I think this (newer) hook is a better way to remember previous values than the old preferences system.